packages/base-styles/_default-custom-properties.scss

```diff
@@ -5,5 +5,4 @@
 	@include admin-scheme(#007cba);
 	--wp-block-synced-color: #7a00df;
 	--wp-block-synced-color--rgb: #{hex-to-rgb(#7a00df)};
-	--wp-bound-block-color: #9747ff;
```

Comment:
As for changing the colors, I think it's okay, but the `base-styles` package itself is published as an npm package, and developers may import it. Although this CSS variable was [recently added](https://github.com/WordPress/gutenberg/pull/59185/files#diff-aab96d21f052689c7fc574a810e44b3bcdf9c52e66aa02a47763ee522c0b75c3R8), I don't think it should be removed. Otherwise, if the developer had defined a style like this, the colors would be lost.

```scss
@import 'node_modules/@wordpress/base-styles/default-custom-properties';

.element {
  color: var(--wp-bound-block-color)
}
```

Reply:
> I don't think it should be removed. Otherwise, if the developer had defined a style like this, the colors would be lost.

What wiggleroom is there for errors, which this clearly was? I'd tend to think that with a proper changelog note it should be possible to deprecate this just like other JS pieces can be deprecated.

Reply:
My approach is to leave the CSS variable and change it to reference the recommended CSS variable.

```scss
:root {
	--wp-block-synced-color: #7a00df;
	// This CSS variable is not used in Gutenberg project,
	// but is maintained for backwards compatibility.
	--wp-bound-block-color: var(--wp-block-synced-color);
}
```

Reply:
Makes sense to me!

Reply:
Added in https://github.com/WordPress/gutenberg/pull/60617/commits/1c09419303e70275f4cdd525a0fa7940fe77ed36